projects/Mallard/src/helpers/files.ts

```diff
@@ -155,15 +156,15 @@ const withPathPrefix = (prefix: string) => (str: string) => `${prefix}/${str}`
 
 export const getLocalIssues = async () => {
     const editionDirectory = await FSPaths.editionDir()
-    const edition = await getSetting('edition')
+    const edition = await getSelectedEditionSlug()
```

Comment:
All `getSelectedEditionSlug` is designed to replace the previous `getSetting('edition')` system. As `selectedEdition` lives in the hook, we need a way of getting this out for non react components. This is why we backup to Async Storage and provide this helper function to get the required slug